### PR TITLE
[xcode11] [HealthKit] b1-b3 bindings

### PR DESCRIPTION
**Pull request by whitneyschmidt**
No b4 diffs.

Issue for `HKCharacteristicTypeIdentifierActivityMoveMode` API unavailable https://github.com/xamarin/maccore/issues/1899

### CODE REVIEW

File: src/healthkit.cs [hunk 1]
```diff
@@ -2840,4 +3004,103 @@ interface HKLiveWorkoutBuilder
...
+	[iOS (13,0)]
+	[Watch (6,0)]
+	[BaseType (typeof (HKSeriesSample))]
+	[DisableDefaultCtor]
+	interface HKHeartbeatSeriesSample : NSSecureCoding {}
+	
+	delegate void HKHeartbeatSeriesBuilderCompletionHandler (bool success, NSError error);
+
+	[Watch (6,0), iOS (13,0)]
+	[BaseType (typeof(HKSeriesBuilder))]
```
Comment (rolfbjarne) [inline src/healthkit.cs]:
```suggestion
	[BaseType (typeof (HKSeriesBuilder))]
```

File: src/healthkit.cs [hunk 2]
```diff
@@ -2840,4 +3004,103 @@ interface HKLiveWorkoutBuilder
...
+
+		[Export ("finishSeriesWithCompletion:")]
+		[Async]
+		void FinishSeries (Action<HKHeartbeatSeriesSample, NSError> completion);
+	}
+
+	delegate void HKHeartbeatSeriesQueryDataHandler (HKHeartbeatSeriesQuery query, double timeSinceSeriesStart, bool precededByGap, bool done, NSError error);
+
+	[Watch (6,0), iOS (13,0)]
+	[BaseType (typeof(HKQuery))]
```
Comment (rolfbjarne) [inline src/healthkit.cs]:
```suggestion
	[BaseType (typeof (HKQuery))]
```

File: tests/introspection/iOS/iOSApiProtocolTest.cs [hunk 3]
```diff
@@ -287,7 +287,9 @@ protected override bool Skip (Type type, string protocolName)
 				case "ARSkeleton2D": // Conformance not in headers
 				case "ARCollaborationData":  // Conformance not in headers
 				case "UIFontPickerViewControllerConfiguration": // Conformance not in headers
-					return true;
+				case "HKAudiogramSample": // Conformance not in headers
+				case "HKAudiogramSensitivityPoint": // Conformance not in headers
+                            return true;
```
Comment (rolfbjarne) [inline tests/introspection/iOS/iOSApiProtocolTest.cs]: Indentation is off here (tabs/space mixup?)

File: tests/introspection/iOS/iOSApiProtocolTest.cs [hunk 4]
```diff
@@ -512,7 +514,8 @@ protected override bool Skip (Type type, string protocolName)
 				case "ARSkeleton2D": // Conformance not in headers
 				case "ARCollaborationData":  // Conformance not in headers
 				case "UIFontPickerViewControllerConfiguration":  // Conformance not in headers
-					return true;
+				case "HKAudiogramSensitivityPoint":  // Conformance not in headers
+                            return true;
```
Comment (rolfbjarne) [inline tests/introspection/iOS/iOSApiProtocolTest.cs]: Indentation is off here (tabs/space mixup?)

File: tests/introspection/iOS/iOSApiProtocolTest.cs [hunk 5]
```diff
@@ -659,7 +662,10 @@ protected override bool Skip (Type type, string protocolName)
 				case "HMAction":
 				case "HMCharacteristicWriteAction":
 				case "MLPredictionOptions":
-					return true;
+                case "HKCumulativeQuantitySample": // Conformance not in headers
+                case "HKDiscreteQuantitySample": // Conformance not in headers
+                case "HKAudiogramSample": // Conformance not in headers
+                            return true;
```
Comment (rolfbjarne) [inline tests/introspection/iOS/iOSApiProtocolTest.cs]: Indentation is off here (tabs/space mixup?)

File: src/healthkit.cs [hunk 6]
```diff
@@ -989,7 +1042,17 @@ interface HKCharacteristicType {
 	[Abstract] // The HKSampleType class is an abstract subclass of the HKObjectType class, used to represent data samples. Never instantiate an HKSampleType object directly. Instead, you should always work with one of its concrete subclasses [...]
 #endif
 	interface HKSampleType {
+		[Export ("isMaximumDurationRestricted")]
```
[the remark this answers is not in the document]
Comment (joconte) [reply src/healthkit.cs]: According to Apple doc : 
```C#
[iOS (13,0), Watch (6,0)]
```
https://developer.apple.com/documentation/healthkit/hksampletype/3327019-ismaximumdurationrestricted

**Comment by whitneyschmidt** [comment]
build

**Comment by whitneyschmidt** [comment]
build

**Pull request closed without merging**